src/SumoCoders/FrameworkCoreBundle/Resources/assets/js/Framework/Form.js

```diff
@@ -4,6 +4,7 @@ const locale = new Locale()
 
 export class Form {
   constructor (form) {
```

Comment:
What's this form in the constructor? I don't see it getting used anywhere

Reply:
Used to be used in SearchForm I think and for datefields, but I think this has been removed in the ES6 update, so might be obsolete now.